signalfx-java/src/main/java/com/signalfx/metrics/connection/HttpDataPointProtobufReceiverFactory.java

```diff
@@ -42,6 +48,11 @@ public HttpDataPointProtobufReceiverFactory setMaxRetries(int maxRetries) {
         return this;
     }
 
+    public HttpDataPointProtobufReceiverFactory setNonRetryableExceptions(List<Class<? extends IOException>> clazzes) {
+        this.nonRetryableExceptions = clazzes;
```

Comment:
nit: prefer copying the list contents rather than holding reference (to prevent future mutability).